cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -86,7 +88,59 @@ def filepath_list(reporter, volume):
     return sorted(glob(glob_path))
 
 
-def check_for_match(new_case, possibilities):
+def get_cosine(vec1: Counter, vec2: Counter) -> float:
```

Comment:
Why not use the get_cosine_similarity function that's over in string_diff? Looks like you copy/pasted it? Or maybe you found this code in the same place I did whenever I found it? Anyway, probably better to use the other one and not have two?

Reply:
I did not find it in CL - but in medium or stack or somewhere else.  I'll adjust to using it.  Oddly enough it doesn't look like it's used anywhere else in CL

Reply:
this should be fixed.